tgui/packages/tgui/interfaces/Mecha/ModulesPane.tsx

```diff
@@ -477,16 +490,46 @@ const SnowflakeSyringe = (props) => {
         />
       </LabeledList.Item>
       <LabeledList.Item label={'Reagent control'}>
+        {analyzed_reagents.map((reagent) => (
+          <LabeledList.Item key={`${reagent}`} label={`${reagent}`}>
```

Comment:
is reagent a string?
```suggestion
          <LabeledList.Item key={reagent} label={reagent}>
```
i'm assuming reagent is an object though, in which case you need some sort of unique identifier, reagent.name or otherwise